symengine/polys/basic_conversions.h

```diff
@@ -133,6 +133,39 @@ class BasicToUPolyBase : public BaseVisitor<V>
         dict = P::container_from_dict(gen, {{0, typename P::coef_type(i)}});
     }
 
+    template <
+        typename Poly,
+        typename
+        = enable_if_t<((std::is_base_of<UIntPolyBase<typename P::container_type,
+                                                     P>,
+                                        P>::value
+                        and std::is_base_of<UIntPolyBase<
+                                                typename Poly::container_type,
+                                                Poly>,
+                                            Poly>::value)
+                       or (std::is_base_of<URatPolyBase<
+                                               typename P::container_type, P>,
+                                           P>::value
+                           and (std::is_base_of<UIntPolyBase<typename Poly::
+                                                                 container_type,
+                                                             Poly>,
+                                                Poly>::value
+                                or std::is_base_of<URatPolyBase<
+                                                       typename Poly::
+                                                           container_type,
+                                                       Poly>,
+                                                   Poly>::value))
+                       or (std::is_same<P, UExprPoly>::value
+                           and std::is_base_of<UPolyBase<typename Poly::
+                                                             container_type,
+                                                         Poly>,
+                                               Poly>::value))
+                      and not std::is_same<Poly, GaloisField>::value>>
```

Comment:
You should try locally. Waiting for travis to check each time is not going to work.
The formatting fails because you are exceeding the number of characters per line